easybuild/easyconfigs/a/ATK/ATK-2.27.1-foss-2017b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'ATK'
+version = '2.27.1'
+
+homepage = 'https://developer.gnome.org/ATK/stable/'
+description = """
+ ATK provides the set of accessibility interfaces that are implemented by other
+ toolkits and applications. Using the ATK interfaces, accessibility tools have
+ full access to view and control running applications.
+"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['673a953987b301ab1e24e7d11677b6e7ba3226411a168449ba946765b6d44297']
+
+builddependencies = [
+    ('GObject-Introspection', '1.53.5', '-Python-2.7.14'),
+]
```

Comment:
@boegel - perhaps one for the style checker in future is whether to have (as I prefer) a line in between `builddependencies` and `dependencies`. It's not consistent in this PR.

Reply:
Yes, that would certainly help making things more consistent.

A bigger one is checking the order of easyconfig parameters though, see https://github.com/easybuilders/easybuild-framework/pull/2165